### ISSUE TITLE
Typo in Step 121 of Create RPG game project

### ISSUE THREAD

**Issue by Sboonny** [freeCodeCamp/freeCodeCamp#47660]
### Describe the Issue

There is a typo in Step 121, the word `to` is missing in the sentence.

Here is the typo

https://github.com/freeCodeCamp/freeCodeCamp/blob/f80e34a9509f3208c01524ea25dd573fcacbe760/curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62a8e49f4df7af5ae2d7a616.md#L14

it should be 

> You should use compound assignment to modify the `monsterHealth` variable.

### Additional context

If you would like to fix this issue, please make sure you read [our guidelines for contributing](https://contribute.freecodecamp.org/#/), we prioritize contributors following the instructions in our guides. Join us in [our chat room](https://discord.gg/PRyKn3Vbay) or [the forum](https://forum.freecodecamp.org/c/contributors/3) if you need help contributing, our moderators will guide you through this.

Sometimes we may get more than one pull requests. We typically accept the most quality contribution, followed by the one that is made first.

Happy contributing.

**Comment by Shivashish-rwt** [comment]
I would love to fix the issue can u assign it to me??

**Comment by utkershrajvenshi** [comment]
I would like to fix this issue as part of Hacktoberfest '22. Could you please assign it to me?

**Comment by utkershrajvenshi** [comment]
@Sboonny if I open PR right now, will it count for Hacktoberfest?

**Comment by Sboonny** [comment]
don't think so, it shouldn't have started yet

**Comment by utkershrajvenshi** [comment]
Yes, it has not. Anyways I will fix it..

**Comment by utkershrajvenshi** [comment]
@Sboonny getting this error while modifying the .md file. Suggestions?
`ERROR 

fcc-replace-challenge
  attempting to replace 15-javascript-algorithms-and-data-structures-22/learn-bas
ic-javascript-by-building-a-role-playing-game/62a8e49f4df7af5ae2d7a616.md

  2022/javascript-algorithms-and-data-structures is not a valid superblock`

**Comment by ieahleen** [comment]
You changed only this file?

To see this part of the curriculum you need the `.env` file, you need to set upcoming changes to `true`.
This line:
https://github.com/freeCodeCamp/freeCodeCamp/blob/f80e34a9509f3208c01524ea25dd573fcacbe760/sample.env#L60
to true

You will also need to rebuild the client — you may be good to go with `npm run clean-and-develop`

**Comment by utkershrajvenshi** [comment]
Ok. Will incorporate this suggestion and get back to you. Thanks!

**Comment by utkershrajvenshi** [comment]
Makes sense. I will interact in contributors chatroom at Discord in case of more setbacks.
